src/util/options.rs

```diff
@@ -716,7 +725,9 @@ options! {
     // Default to a fixed heap size of 0.5x physical memory.
     gc_trigger     :        GCTriggerSelector    [env_var: true, command_line: true] [|v: &GCTriggerSelector| v.validate()] = GCTriggerSelector::FixedHeapSize((crate::util::memory::get_system_total_memory() as f64 * 0.5f64) as usize),
     // Enable transparent hugepage support via madvise (only Linux is supported)
-    transparent_hugepages: bool                  [env_var: true, command_line: true]  [|v: &bool| !v || cfg!(target_os = "linux")] = false
+    transparent_hugepages: bool                  [env_var: true, command_line: true]  [|v: &bool| !v || cfg!(target_os = "linux")] = false,
+    // Enable 35-bit heap address space (normally for compressed pointers)
+    use_35bit_address_space: bool                [env_var: true, command_line: true]  [always_valid] = false
```

Comment:
Can we make this an enum to choose from different sets of vm constants

Reply:
This option is no longer used in the PR, and can be removed.

Reply:
Removed